easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-v1706-cleanup.patch

```diff
@@ -0,0 +1,406 @@
+--- OpenFOAM-v1706.orig/applications/utilities/mesh/manipulation/setSet/Allwmake	2017-06-30 10:50:14.000000000 +0200
```

Comment:
it's good practice to claim authorship of the patches and explain what they are supposed to do.